workspace/utilities/lib/menu-admin.xsl

```diff
@@ -49,7 +49,7 @@
 					<xsl:call-template name="element">
 						<xsl:with-param name="attr" select="$computed-attr-title" />
 						<xsl:with-param name="content">
-							<span>Admin</span>
+							<span>Admin — <xsl:value-of select="/data/events/login-info/name" /></span>
```

Comment:
Can we display only the first name ?

Reply:
I display the first word only, which should be the first name almost all the time. I don't have another way to split the information. 

> The only missing piece is @PascalPiche's comment
>> Move most of the css in class behavior as much as possible
Was already fixed in a previous commit